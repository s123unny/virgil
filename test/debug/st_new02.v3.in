b ../core/st_new02.v3 5
run
bt
info l
c
q
