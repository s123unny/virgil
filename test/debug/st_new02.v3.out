breakpoint|../core/st_new02.v3 5|0
stopOnBreakpoint
bt|main|../core/st_new02.v3|5
variable|0|x|0|int|false
end
result|!DivideByZeroException
	in st_new02b.new() [../core/st_new02.v3 @ 3:45]
	in main() [../core/st_new02.v3 @ 5:29]

